cmd/greenhouse/controllers.go

```diff
@@ -122,3 +124,27 @@ func startClusterHeadscaleAccessReconciler(name string, mgr ctrl.Manager) error
 		RenewRemoteClusterBearerTokenAfter:       renewRemoteClusterBearerTokenAfter,
 	}).SetupWithManager(name, mgr)
 }
+
+func startTeamMembershipUpdaterReconciler(name string, mgr ctrl.Manager) error {
+	scimBaseURL := os.Getenv(scimBaseURLEnvKey)
+	if scimBaseURL == "" {
+		setupLog.Error(nil, fmt.Sprintf("%s env needs to be set for running the scim client", scimBaseURLEnvKey))
```

Comment:
I think if you use `setupLog.Error...` it is going to throw an error and the whole controller wont start. I would recommend to use `setupLog.Info....`. @IvoGoman what do you think ?
```suggestion
		setupLog.Info("Scim client env needs to be set for running the scim client","env", scimBaseURLEnvKey)
```

Reply:
@kengou I think you're right. As with the other controllers, I can log it as Info with WARN tag, another log about skipping the controller, and then returning. I'll make the changes.